### PR TITLE
feat: added privacy policy to homepage

### PR DESCRIPTION
**Pull request by Sonichigo**
- Under Copyright © 2023 Keploy Inc. Section Policy Link is added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -764,6 +764,7 @@ <h2 class="mb-30">Sign-up for the newsletter!</h2>
 						<div class="f_title">
 							<h3>Copyright © 2023  Keploy Inc.</h3>
 						</div>
+						<p><a href="https://docs.keploy.io/privacy-policy/" style="color: #f37d36;">Privacy Policy</a></p>
```
Comment (slayerjain) [inline index.html]: please put this below the lower component (`<p>Developers e..</p>`)

**Review by slayerjain — CHANGES_REQUESTED** [review]
need to move the privacy policy further down.

**Comment by slayerjain** [comment]
@Sonichigo also something else todo is that the privacy policy should be hosted here (eg: `https://keploy.io/privacy-policy`)

File: index.html [hunk 2]
```diff
@@ -766,6 +766,7 @@ <h3>Copyright © 2023  Keploy Inc.</h3>
 						</div>
 						<p>Developers experience for e2e testing. Toolkit that creates test-cases and data mocks from API calls, DB queries, etc.</p>
 <!--						<img src="img/cncf-keploy.png" class="w-75" alt="cncf_logo">-->
+						<p><a href="https://docs.keploy.io/privacy-policy/" style="color: #f37d36;">Privacy Policy</a></p>
```
Comment (nehagup) [inline index.html]: IMO this should be directing to privacy.html

**Comment by slayerjain** [comment]
Thanks @Sonichigo LGTM!

**Review by slayerjain — APPROVED** [review]

**Pull request merged**